src/app/tests/suites/certification/Test_TC_TBRM_2_5.yaml

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) 2024 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name:
+    "[TC-TBRM-2.5] Verify network configuration of a Thread Border Router
+    endpoint that support also Secondary Network Interface device type"
+
+PICS:
+    - TBRM.S
+    - CNET.S.F01
+
+config:
+    nodeId: 0x12344321
+    cluster: Thread Border Router Management
+    endpoint: 1
+    PIXIT.TBRM.THREAD_ACTIVE_DATASET:
+        type: octet_string
+        defaultValue: "hex:0e08000000000001000035060004001fffe00708fddeb3d807f0cc3f04104b575fabba5fdb5917c8267eca0674740c0402a0f7f8000300001901025b3502085b35dead5b35beef030435623335051000112233445566778899aabbccddeeff"
+
+tests:
+    - label: "Wait for the commissioned device to be retrieved"
+      cluster: DelayCommands
+      command: WaitForCommissionee
+      arguments:
+          values:
+              - name: nodeId
+                value: nodeId
+
+    # Step 1
+    - label: "TH sends a GetActiveDatasetRequest command to the DUT"
+      command: GetActiveDatasetRequest
+      response:
+          values:
+              - name: Dataset
+                value: PIXIT.TBRM.THREAD_ACTIVE_DATASET
+                constraints:
+                    type: octet_string
+
+    # Step 2
+    - label: "TH read MaxNetworks attribute from Network Commissioning Cluster"
+      cluster: NetworkCommissioning
+      command: readAttribute
+      attribute: MaxNetworks
+      response:
+          value: 1
+
+    # Step 3
+    - label: "TH reads the InterfaceEnabled attribute from the DUT"
+      command: readAttribute
```

Comment:
This all assumes the same endpoint hosts TBRM and Network Commissioning, which is not necessarily the case.

Reply:
But that is what we are saying in the spec PR - that the same endpoint is hosting both device IDs (for the condition of the normative requirements to apply).